packages/medusa/src/utils/build-query.ts

```diff
@@ -120,6 +122,9 @@ function buildWhere<TWhereKeys extends object, TEntity>(
           case "gte":
             where[key].push(MoreThanOrEqual(objectValue))
             break
+          case "contains":
```

Comment:
thought(non-blocking): We also include `equals`, `ends_with`, and `starts_with` in the params, but we are only adding query-building support for `contains`. Would it make sense to add those in now? Seems like a low-hanging fruit to pick

Reply:
though equals is already taken care of by the syntax `description=try-on` which looks for an exact match

Reply:
With that in mind, should we remove `equals` and add `ends_with` and `starts_with` to build query?